vhd/release-notes/aks-windows-2019-containerd/2019-datacenter-core-ctrd-17763.5329.231230.txt

```diff
@@ -0,0 +1,94 @@
+﻿Build Number: 2024.010.2
+Build Id:     85482297
+Build Repo:   
+Build Branch: users/honcao/buildvhd3

```

Comment:
Ideally the vhd is built off of master branch, not sure if it's necessary.

Reply:
based on my understanding of the pipeline, it supports to build VHD from any branch.
And we and refresh the build number later after the PR is merged.